content/docs/cml/self-hosted-runners.md

```diff
@@ -83,27 +83,27 @@ you do it your way :)
-| `--cloud-type`           | Instance type.                                                                                                  | Choices: `m`, `l`, `xl`. Also supports native types like i.e. `t2.micro`                                       |
-| `--cloud-gpu`            | GPU type.                                                                                                       | Choices: `nogpu`,`k80`,`tesla`                                                                                 |
-| `--cloud-hdd-size`       | HDD size in GB.                                                                                                 | Accepts integer values                                                                                         |
-| `--cloud-ssh-private`    | Your private RSA SSH key. If not provided will be generated by the Terraform-provider-Iterative.                | Accepts string                                                                                                 |
-| `--cloud-spot`           | Request a spot instance                                                                                         | Boolean                                                                                                        |
-| `--cloud-spot-price`     | Spot max price. If not specified it takes current spot bidding pricing.                                         | Default: `-1`                                                                                                  |
-| `--cloud-startup-script` | Run the script provided in-line during the instance initialization. Expects a Base64-encoded Linux shell script | String                                                                                                         |
-| `-h`                     | Show help                                                                                                       | Boolean                                                                                                        |
+| Argument                 | Type                                                                  | Default                                                    | Description                                                                                                                  |
+| ------------------------ | --------------------------------------------------------------------- | ---------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------- |
+| `--version`              | Flag                                                                  | Unset                                                      | Show version number and exit.                                                                                                |
```

Comment:
I'd avoif "unset" since it's not clear what that means. Maybe just a `-` or "N/A" is more obvious:

```suggestion
| `--version`              | Flag                                                                  | -                                                      | Show version number and exit (ignoring all other options).                                                                                                |
```

And also, aren't all the args/options "flags"? Some flags just happen to accept an argument themselves. If so I would go back to `Boolean` here for now. I'll lyk if I can think of a better term... ⌛

Reply:
> _I'd avoid "unset" since it's not clear what that means. Maybe just a `-` or "N/A" is more obvious._

I would prefer to avoid the hyphen because of its `stdin` meaning on many command-line tools. Maybe we can use an [em dash](https://en.wikipedia.org/wiki/Dash#Em_dash) without backticks <kbd>—</kbd> or just leave them blank.

> _And also, aren't all the args/options "flags"? Some flags just happen to accept an argument themselves. If so I would go back to `Boolean` here for now. I'll lyk if I can think of a better term..._ ⌛

Yes, I was thinking of boolean flags (like bit flags), but the word _flag_ is also overloaded to mean _option_ even with arguments. Fortunately enough, we can get rid of the whole **Type** column.

Reply:
> I would prefer to avoid the hyphen

Sure. I changed it to "N/A" (does not apply) in my suggestion above.

> Fortunately enough, we can get rid of the whole Type column.

Agreed. I removed that cell in my suggestion ☝️ too.

Reply:
Resolved with https://github.com/iterative/dvc.org/pull/2321/commits/ca804af226acd6100e23d25430d39b87277ef6f8